packages/flutter/lib/src/widgets/scroll_activity.dart

```diff
@@ -49,6 +49,12 @@ abstract class ScrollActivityDelegate {
   /// Terminate the current activity and start a ballistic activity with the
   /// given velocity.
   void goBallistic(double velocity);
+
+  /// Update the ballistic animation when layout is change.
+  /// Instead of restarting the animation while fling.
+  /// [initVelocity] is the velocity when ballistic animation start
+  /// [initPosition] is the position when ballistic animation start
```

Comment:
Is this what you mean?

```suggestion
  /// Update the ballistic animation instead of restarting  it, for example as the result of a layout change after a flinging gesture.
  ///
  /// The [initVelocity] and [initPosition] refer to the starting values of the new ballistic animation.
```

Reply:
Yes .Thanks for helping me fix my grammar